### DISCUSSION TITLE
Allow setting of filter for public code after first setup

### DISCUSSION THREAD

**Discussion by dkbast** [community/community#35465] [Copilot Conversations] [answered]
Hi - I just got a bunch of suggestions from public code which I would like to avoid. How can I invoke the filter after first setup?

From the marketing page (https://github.com/features/copilot): 
What can I do to reduce GitHub Copilot’s suggestion of code that matches public code?
We built a filter to help detect and suppress the rare instances where a GitHub Copilot suggestion contains code that matches public code on GitHub. You have the choice to turn that filter on or off during setup. With the filter on, GitHub Copilot checks code suggestions with its surrounding code for matches or near matches (ignoring whitespace) against public code on GitHub of about 150 characters. If there is a match, the suggestion will not be shown to you. We plan on continuing to evolve this approach and welcome feedback and comment

**Comment by davecheney** [comment, marked as answer]
You can change this setting at https://github.com/settings/copilot
